sbpy/data/phys.py

```diff
@@ -9,18 +9,29 @@
 created on June 04, 2017
 """
 
+import warnings
+
 from collections import OrderedDict
 
 from numpy import ndarray, array, isnan, nan
 import astropy.units as u
+import astropy.constants as con
 from astroquery.jplsbdb import SBDB
+from astroquery.jplspec import JPLSpec
 
 from .core import DataClass
 from .. import bib
+from ..exceptions import SbpyWarning
 
 __all__ = ['Phys']
 
 
+class JPLSpecQueryFailed(SbpyWarning):
```

Comment:
I looked at the code raising this exception, and they are fatal errors, i.e., the code cannot complete as requested.  Instead of `SbpyWarning` I recommend using `SbpyException`.